pkg/karmadactl/promote/promote.go

```diff
@@ -19,6 +19,7 @@ package promote
 import (
 	"context"
 	"fmt"
+	"github.com/karmada-io/karmada/pkg/resourceinterpreter/customized/webhook/request"
```

Comment:
```shell
diff --git a/pkg/karmadactl/promote/promote.go b/pkg/karmadactl/promote/promote.go
index 1f8b8fdf..5471e9c3 100644
--- a/pkg/karmadactl/promote/promote.go
+++ b/pkg/karmadactl/promote/promote.go
@@ -19,7 +19,6 @@ package promote
 import (
        "context"
        "fmt"
-       "github.com/karmada-io/karmada/pkg/resourceinterpreter/customized/webhook/request"
        "os"
 
        "github.com/spf13/cobra"
@@ -50,6 +49,7 @@ import (
        "github.com/karmada-io/karmada/pkg/karmadactl/util"
        "github.com/karmada-io/karmada/pkg/resourceinterpreter/customized/declarative"
        "github.com/karmada-io/karmada/pkg/resourceinterpreter/customized/webhook"
+       "github.com/karmada-io/karmada/pkg/resourceinterpreter/customized/webhook/request"
        "github.com/karmada-io/karmada/pkg/resourceinterpreter/default/native"
        "github.com/karmada-io/karmada/pkg/resourceinterpreter/default/native/prune"
        "github.com/karmada-io/karmada/pkg/resourceinterpreter/default/thirdparty"
```
 Hi @zhy76 The goimport is not formatted and therefore also fails ```ci```

Reply:
Fixed, Thanks~